app/code/Magento/Backend/view/adminhtml/templates/widget/grid.phtml

```diff
@@ -48,6 +48,7 @@ $numColumns = sizeof($block->getColumns());
                 <?php else: ?>
                     <?= $block->getMainButtonsHtml() ? '<div class="admin__filter-actions">' . $block->getMainButtonsHtml() . '</div>' : '' ?>
                 <?php endif; ?>
+                <?php $block->getCollection()->load() ?>
```

Comment:
We don't need to load collection forcible

Reply:
@naydav we need to execute code located in _beforeLoad() method.

Reply:
@naydav  fixed.